app/src/ui/history/commit-summary.tsx

```diff
@@ -226,12 +244,12 @@ export class CommitSummary extends React.Component<ICommitSummaryProps, ICommitS
             </li>
 
             <li className='commit-summary-meta-item'
-              title={shortSHA} aria-label='SHA'>
+              title="Copy SHA to clipboard" aria-label='SHA'>
               <span aria-hidden='true'>
                 <Octicon symbol={OcticonSymbol.gitCommit} />
               </span>
 
-              {url ? <LinkButton uri={url}>{shortSHA}</LinkButton> : shortSHA}
+              <LinkButton onClick={() => clipboard.writeText(longSHA)}>{shortSHA}</LinkButton>
```

Comment:
Could we move this into a bound method on the component instead? We have a tslint rule to prevent this but unfortunately it doesn't seem like it's catching this one :/

You can follow the same pattern as `onExpand`, i.e. 

```ts
private onCopyShaToClipboard = () => {
  clipboard.writeText(this.props.sha)
}}
```